DOCS-12330 v1 EOL (#242)
mulesoft/docs-api-community-manager: modules/ROOT/pages/update-acm.adoc

To ensure that the community administrator controls how their audience receives community changes, new functionality in upgrades is typically disabled until the community administrator enables it.

> API Community Manager follows the standard support policy for MuleSoft cloud offerings, and only the latest version of API Community Manager 2.x is supported. API Community Manager 1.x reached End of Life (EOL) on August 14, 2021 and is not supported.

Reviewer: Is the standard support policy = supporting only the latest version? Or is the first part of the sentence actually different from the second (following ", and")?

Reviewer: Also...does this mean that if the latest version of ACM is 2.5, neither 2.4 and earlier, and of course 1.x versions, are supported?

Reviewer: If ^ are true, can we use
```suggestion
Because API Community Manager follows the standard support policy for MuleSoft cloud offerings, only the latest version of API Community Manager 2.x is supported. Additionally, API Community Manager 1.x reached End of Life (EOL) on August 14, 2021.
```

Author: Correct, the standard policy is that the latest version (2.5 in the example) is supported, 2.4 and earlier aren't, and 1.x aren't. That edit looks great, and I've updated the PR to use it.
